aries-site/src/pages/components/table.mdx

```diff
@@ -0,0 +1,240 @@
+to solve? A good opportunity for speaking to and learning from end users!
+
+**The Exception**<br />
+In an analytical context, where the goal is to assemble a data set and identify patterns 
+within it, a larger number of columns may be neccessary.
+
+In this context, a Table should facilitate the assembly of the data set and make the set 
+accessible in a tool specifically designed for analysis.
+-->
+
+## Table behaviors & actions
+
+### Sorting
+Column headers behave as a button and perform the sort action. A caret icon indicates 
+the directionality and to which column the sort is currently applied.
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/table/TableSortable.js"
+  docs="https://v2.grommet.io/datatable?theme=hpe#props"
+  height={{ min: 'small' }}
+>
+  <TableSortable />
+</Example>
+
+### Searching and filtering tables
+Complimenatary elements such as search or filters should be guided by the task the user 
+is setting out to accomplish.
+- Looking for a specific record? --> Search.
+- Trying to narrow records down to a set of shared characteristics? --> Filter.
+- Not quite certain for what I am looking, but I know its general characteristics? --> Filter.
+
+[See Searching & Filtering]() for greater detail.
+
+### Selecting single records
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/table/TableSingleSelectExample.js"
+  docs="https://v2.grommet.io/datatable?theme=hpe#props"
+  height={{ min: 'small' }}
+>
+  <TableSingleSelectExample />
+</Example>
+
+### Selecting multiple records & batch actions
+Checkboxes allow for selection of multiple records, which then may apply 
+batch actions via action control buttons or menus. 
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/table/TableMultiSelectExample.js"
+  docs="https://v2.grommet.io/datatable?theme=hpe#props"
+  height={{ min: 'small' }}
+>
+  <TableMultiSelectExample />
+</Example>
+
+### Fixing header row and/or columns
+Fixing or pinning header rows, columns, or footers allow users to track and 
+maintain context across long and wide data sets.
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/table/TableFixedHeaderExample.js"
+  docs="https://v2.grommet.io/datatable?theme=hpe#props"
+  height={{ min: 'small' }}
+>
+  <TableFixedHeaderExample />
+</Example>
+
+### Column summaries & aggregation
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/table/TableSummaryExample.js"
+  docs="https://v2.grommet.io/datatable?theme=hpe#props"
+  height={{ min: 'small' }}
+>
+  <TableSummaryExample />
+</Example>
+
+### Use pagination or infinite scroll with tables
```

Comment:
This section feels a bit disjointed from the rest of the page flow -- maybe this should be moved up towards the other blocks of guidance, unless an example is going to accompany it

Reply:
Agree. I think the desire is to have accompanying examples. I have created an issue for this content, designs, and examples: https://github.com/hpe-design/design-system/issues/1130

Reply:
Awesome!